nipype/interfaces/afni/preprocess.py

```diff
@@ -3108,28 +3134,28 @@ class TSmoothInputSpec(AFNICommandInputSpec):
         desc="Sets the data type of the output dataset", argstr="-datum %s"
     )
     lin = traits.Bool(
-        desc="3 point linear filter: 0.15*a + 0.70*b + 0.15*c"
+        desc=r"3 point linear filter: :math:`0.15\,a + 0.70\,b + 0.15\,c`"
         "[This is the default smoother]",
         argstr="-lin",
     )
     med = traits.Bool(desc="3 point median filter: median(a,b,c)", argstr="-med")
     osf = traits.Bool(
         desc="3 point order statistics filter:"
-        "0.15*min(a,b,c) + 0.70*median(a,b,c) + 0.15*max(a,b,c)",
+        r":math:`0.15\,min(a,b,c) + 0.70\,median(a,b,c) + 0.15\,max(a,b,c)`",
         argstr="-osf",
     )
     lin3 = traits.Int(
-        desc="3 point linear filter: 0.5*(1-m)*a + m*b + 0.5*(1-m)*c"
+        desc=r"3 point linear filter: :math:`0.5\,(1-m)\,a + m\,b + 0.5\,(1-m)\,c"
```

Comment:
Some serious distortion going on here: https://2093-17021985-gh.circle-artifacts.com/0/tmp/src/nipype/doc/_build/html/api/generated/nipype.interfaces.afni.preprocess.html#tsmooth

```suggestion
        desc=r"3 point linear filter: :math:`0.5\,(1-m)\,a + m\,b + 0.5\,(1-m)\,c`"
```

Again, though, I'm not sure that this is good for `TSmooth.help()`.